lib/src/executable/options.dart

```diff
@@ -251,6 +252,8 @@ class ExecutableOptions {
             {null: _options.rest.isEmpty ? null : _options.rest.first});
       } else if (_options.rest.length > 2) {
         _fail("Only two positional args may be passed.");
+      } else if (directories.isNotEmpty) {
+        _fail('Directory "${directories.first}" may not be a positional arg.');
```

Comment:
Does this mean it is valid to envoke `sass input.scss output.css` but not `sass input_dir output_dir`? If so maybe the error message would be more clear if you say something that suggests the `input_dir:output_dir` syntax.